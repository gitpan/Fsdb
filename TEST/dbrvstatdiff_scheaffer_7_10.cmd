# example 7.10 from Scheaffer and McClave
prog='dbrvstatdiff'
args='-h "<=0" x2 sd2 n2 x1 sd1 n1'
cmp='diff -cb '
