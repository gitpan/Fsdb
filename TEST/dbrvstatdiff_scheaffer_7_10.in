#h title x2 sd2 n2 x1 sd1 n1
example7.10 9 35.22 24.44 9 31.56 20.03
