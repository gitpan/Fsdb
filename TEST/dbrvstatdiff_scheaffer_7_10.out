#fsdb title x2 sd2 n2 x1 sd1 n1 diff diff_pct diff_conf_half diff_conf_low diff_conf_high diff_conf_pct_half diff_conf_pct_low diff_conf_pct_high t_test t_test_result t_test_break t_test_break_pct
example7.10	9	35.22	24.44	9	31.56	20.03	0	0	20.455	-20.455	20.455	227.27	-227.27	227.27	0	not-rejected	-20.454525503344	-227.27
#   | dbrvstatdiff -h <=0 x2 sd2 n2 x1 sd1 n1
